packages/amplify/amplify_flutter/test/amplify_test.dart

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-import 'package:amplify_analytics_pinpoint/method_channel_amplify.dart';
+import 'package:amplify_datastore/method_channel_datastore.dart';
```

Comment:
This test is still failing, will ping offline w suggestions.

Reply:
Thanks for the fixes